assets/app/view/game/part/city.rb

```diff
@@ -16,6 +16,7 @@ class City < Base
 
         needs :tile
         needs :city
+        needs :should_render_revenue
```

Comment:
so why is this needed now and the old method not working?

Reply:
This is the first tile we've had with multicity multislot, I didn't want to dup the logic in two places

Reply:
gotcha, did you clean up all the duplicate logic?

Reply:
can you rename it render_revenue or show_revenue ?

Reply:
Yep, I cleared most of it. There's a little safety code I left in. Will rename to show revenue